sys/shell/commands/shell_commands.c

```diff
@@ -272,6 +279,12 @@ const shell_command_t _shell_command_list[] = {
 #endif
 #ifdef MODULE_SUIT_COAP
     { "suit", "Trigger a SUIT firmware update", _suit_handler },
+#endif
+#ifdef MODULE_CRYPTOAUTHLIB
+    { "rc", "Read device's config zone", _read_config },
+    { "lc", "Permanently lock device's config zone", _lock_config},
+    { "ccl", "Check if config zone is locked", _check_lock_config },
+    { "cdl", "Check if data zone is locked", _check_lock_data },
```

Comment:
though short command are nice, the problem here is that `rc`even with the help text does not refer to `cryptoauthlib` in any way. For instance to test this I added `USEPKG += cryptoauthlib` to `examples/default` and when I run `help` in the shell I get:

```
2020-02-12 14:56:08,555 #  help
2020-02-12 14:56:08,557 # Command              Description
2020-02-12 14:56:08,561 # ---------------------------------------
2020-02-12 14:56:08,564 # reboot               Reboot the node
2020-02-12 14:56:08,570 # ps                   Prints information about running threads.
2020-02-12 14:56:08,573 # random_init          initializes the PRNG
2020-02-12 14:56:08,578 # random_get           returns 32 bit of pseudo randomness
2020-02-12 14:56:08,583 # rtc                  control RTC peripheral interface
2020-02-12 14:56:08,587 # ifconfig             Configure network interfaces
2020-02-12 14:56:08,593 # txtsnd               Sends a custom string as is over the link layer
2020-02-12 14:56:08,598 # saul                 interact with sensors and actuators using SAUL
2020-02-12 14:56:08,602 # rc                   Read device's config zone
2020-02-12 14:56:08,608 # lc                   Permanently lock device's config zone
2020-02-12 14:56:08,612 # ccl                  Check if config zone is locked
2020-02-12 14:56:08,616 # cdl                  Check if data zone is locked
```

Admittedly, this is more a problem with the RIOT shell, which doesn't have something like command sections. However, this could be somewhat enhanced by introducing a "super" command, e.g. `cryptoauth` which then handles subcommands like `cryptoauth read config`, `cryptoauth lock <zone>` or something like that.